app/assets/javascripts/discourse/app/widgets/post.js

```diff
@@ -352,22 +353,25 @@ createWidget("post-date", {
   tagName: "div.post-info.post-date",
 
   html(attrs) {
-    const attributes = { class: "post-date" };
-    let date;
-    if (attrs.wiki && attrs.lastWikiEdit) {
```

Comment:
Was that test there for a reason or just out of precaution?

Reply:
I think just out of precaution. I looked at the serializer and last_wiki_edit is present only if it is a wiki: https://github.com/discourse/discourse/blob/main/app/serializers/post_serializer.rb#L503-L507.